phpunit/phpunit/4.7/phpunit.xml.dist

```diff
@@ -9,7 +9,11 @@
 >
     <php>
         <ini name="error_reporting" value="-1" />
-        <server name="KERNEL_CLASS" value="App\Kernel" />
+        <env name="KERNEL_CLASS" value="App\Kernel" />
```

Comment:
Why using `env` here when we just started checking  `$_SERVER` (see e.g. #156)?

Reply:
because it's the correct concept, and maps to the .env file
_SERVER mixes several sources, so it's confusing

Reply:
But the difference if I don't miss anything is that PHPUnit unline the Dotenv component does not populate the `$_SERVER` array with this config.

For example, take this config:

```xml
    <php>
        <server name="foo" value="bar" />
        <env name="foobar" value="baz" />
    </php>
```

It ends up with this output:

```
(
    [DOCUMENT_ROOT] => 
    [PATH_TRANSLATED] => ./phpunit
    [PHP_SELF] => ./phpunit
    [PWD] => /Users/christian/symfony
    [REQUEST_TIME] => 1504717866
    [REQUEST_TIME_FLOAT] => 1504717866.6028
    [SCRIPT_FILENAME] => ./phpunit
    [SCRIPT_NAME] => ./phpunit
    [TMPDIR] => /var/folders/9_/wkjkz_ms2hn7rkr4vm4b68t00000gn/T/
    [USER] => christian
    [_] => ./phpunit
    [argc] => 3
    [argv] => Array
        (
            [0] => ./phpunit
            [1] => --colors=always
            [2] => src/Symfony/Component/Filesystem/
        )

    [foo] => bar
)
Array
(
    [foobar] => baz
)
```

Produced by:

```php
ksort($_SERVER);
print_r($_SERVER);
ksort($_ENV);
print_r($_ENV);
```

Reply:
That's stupid :)
Anyway, since `getenv('foo')` returns only from `<env>`, I think we have no choice but use it.

Reply:
Also this line would not meet its goal because: https://github.com/symfony/symfony/blob/d102fc08e48451aee932a8138d9cfaa726cee288/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php#L117.

Reply:
@yceruto good catch, can you send a PR to fix this? it should work!

Reply:
Not sure what I should fix here? you mean check both vars `$_SERVER` and `$_ENV` in `KernelTestCase`?

Reply:
yes, that's what I mean